tests/unit/Storage/Compression/Algorithms/GZIPTest.php

```diff
@@ -2,7 +2,7 @@
 
```

Comment:
We can remove all the duplicated test files. We can rely on the tests that runs at utopia-php/storage

Reply:
Got it

Reply:
Removed all the storage tests!